tests/test_add_reverb.py

```diff
@@ -4,12 +4,32 @@
 
 input_signal = np.genfromtxt('tests/data/bark.csv', dtype = 'float32')
 
+'''
+Test that TypeError is raised for invalid input_signal argument.
+'''
+def signal_format():
```

Comment:
In pytest, test functions have to start with `test_`, for example: `test_signal_format()`. Otherwise pytest won't run them as tests.